webpack/scenes/Organizations/OrganizationActions.js

```diff
@@ -49,6 +49,9 @@ export const saveOrganization = (extendedParams = {}) => (dispatch) => {
         type: SAVE_ORGANIZATION_SUCCESS,
         response: data,
       });
+
+      // TODO: Necessary because of https://projects.theforeman.org/issues/26420
+      dispatch(loadOrganization());
```

Comment:
do you need to run this with `dispatch`? It seems like `loadOrganization`, doesn't return an object to dispatch, but rather dispatches actions itself. Wouldn't just `loadOrganization()` work here?

Reply:
I don't think it will work since the `loadOrganization` needs the `dispatch` method.
`dispatch(loadOrganization())` is like doing `loadOrganization()(dispatch)`

Reply:
ah, ya I see that now, thanks for explaining!